db/compaction/compaction_job.cc

```diff
@@ -1564,7 +1565,8 @@ Status CompactionJob::OpenCompactionOutputFile(
                            &syncpoint_arg);
 #endif
   Status s;
-  IOStatus io_s = NewWritableFile(fs_, fname, &writable_file, file_options_);
+  IOStatus io_s = NewWritableFile(db_options_.fs.get(), fname, &writable_file,
```

Comment:
Shouldn't we pass the right pointer from ```fs_``` instead of ```db_options_.fs```? Otherwise, this won't get traced.

Reply:
Right now, this is not getting traced.  But since its a one line function to trace FileSystem::NewWritableFile so I will pass the FileSystemPtr pointer.